Formula/f/foldseek.rb

```diff
@@ -0,0 +1,50 @@
+class Foldseek < Formula
+  desc "Search your protein structures in seconds"
+  homepage "https://github.com/steineggerlab/foldseek"
+  url "https://github.com/steineggerlab/foldseek/archive/8-ef4e960.tar.gz"
+  version "8-ef4e960"
+  sha256 "c74d02c4924d20275cc567783b56fff10e76ed67f3d642f53c283f67c4180a1e"
+  license "GPL-3.0-or-later"
+  head "https://github.com/steineggerlab/foldseek.git", branch: "master"
+
+  depends_on "cmake" => [:build, :test]
+  depends_on "rust" => :build
+
+  uses_from_macos "bzip2"
+  uses_from_macos "zlib"
+
+  on_macos do
+    depends_on "libomp"
+  end
+
+  resource "homebrew-testdata" do
+    url "https://raw.githubusercontent.com/steineggerlab/foldseek/master/example/d1asha_"
+    sha256 "b4ec14f5decc94b5363b3414db4d25e3e09039c7a6fbb585041730dcf3cc1fd8"
+  end
+
+  def install
+    # fix "DSO missing from command line" error
+    ENV.append "LDFLAGS", "-Wl,--copy-dt-needed-entries" if OS.linux?
```

Comment:
Can this be reported upstream?

Reply:
It seems that this environment variable was not needed, although required in my environment (CentOS7) for some reason. Anyway, thanks for your help.